pandas/tests/groupby/test_grouping.py

```diff
@@ -628,6 +628,40 @@ def test_groupby_empty(self):
         # check name
         assert s.groupby(s).grouper.names == ["name"]
 
+    def test_groupby_level_index_value_all_na(self):
+        # issue 20519
+        df = pd.DataFrame([["x", np.nan, 10]], columns=["A", "B", "C"]).set_index(
+            ["A", "B"]
+        )
+        result = df.groupby(level=["A", "B"]).sum()
+        expected = DataFrame(
```

Comment:
Hmm just thinking out loud but why do we think this output is valuable? Read through issue but I wonder if this shouldn't raise instead. Since column `B` is just an NA value - what is the value in making that "groupable"?

Reply:
@WillAyd 
Of course, NA values are non groupable value.  So the logic which is what i added fills grouper variable with numpy nan considered NA value in Pandas. If some values are missing(not all values are missing) in a level of multiindex, fill another value in a level which is not NA value. 
Like this, I want to fill garbage value because there is no non-NA value. In this case, numpy nan.
Consequently, I want a consistency. For example, 
df = pd.DataFrame([["a", None, "x", 10], [None, "y", None, 20]], columns=["A", "B", "C", "D"]).set_index(["A","B","C"])
result = df.groupby(level=["A","B"]).sum()
this result variable is empty dataframe before what i made is added. So, what i added also makes sense?

Reply:
Yea understood on the consistency but shouldn't these both raise then? Not sure why we would want to arbitrarily fill values for the user - @TomAugspurger I think you might have a differing opinion

Reply:
I'm happy to be overruled here @WillAyd. I don't think I understand the original issue that well.

Reply:
Hmm OK I think I'm back on board with this after thinking it over some more. This at the very least matches what you would get in more simple cases too:

```python
>>> df = pd.DataFrame({"A": [1, 1], "B": [np.nan, np.nan]})
>>> df.groupby("B").sum()
Empty DataFrame
Columns: [A]
Index: []
```